service/project_service/upload.go

```diff
@@ -0,0 +1,30 @@
+package service
+
+import (
+	"mime/multipart"
+
+	"github.com/guregu/null"
+
+	"github.com/torinos-io/api/type/model"
+)
+
+// UploadRequest holds uploaded files
+type UploadRequest struct {
+	CartfileContent    *multipart.FileHeader `json:"cartfile_content"`
```

Comment:
Ideally, `*multipart.FileHeader` is not preferable because it somehow have a concern of http.
It should be either `[]byte` or `io.Reader`.